tools/linter/shellcheck/__init__.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import shutil
+import sys
+import os
+
+
+from tools.linter.utils import run_cmd, CommandResult
+from tools.linter.lint import Linter
+
+
+class Shellcheck(Linter):
+    name = "shellcheck"
+    exe = shutil.which(name)
+    options = argparse.Namespace(
+        paths=[
+            ".extracted_scripts",
+            ".jenkins/pytorch"
+        ],
+        glob=["*.sh"],
```

Comment:
`paths` and `glob` can probably be merged, this might make it easier to understand what's going on since it takes 2 filtering steps (by path, then glob) and combines it all into 1